src/dotnet/commands/dotnet-migrate/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.Migrate
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Migrate Command";
+
+        public const string AppDescription = "Command used to migrate project.json projects to msbuild";
+
+        public const string CmdProjectArgument = "PROJECT_JSON/GLOBAL_JSON/PROJECT_DIR";
+        public const string CmdProjectArgumentDescription = 
+        @"The path to 
+         - a project.json file to migrate.
+        or
+         - a global.json file, it will migrate the folders specified in global.json.
+        or
+         - a directory to migrate, it will recursively search for project.json files to migrate.
+        Defaults to current directory if nothing is specified.";
+
```

Comment:
This does not match the original string, as here there is extra indentation. We need to move everything to the left-most column.